duckdb.opam

```diff
@@ -10,8 +10,16 @@ homepage: "https://github.com/username/reponame"
 doc: "https://url/to/documentation"
 bug-reports: "https://github.com/username/reponame/issues"
 depends: [
-  "dune" {>= "3.17"}
-  "ocaml"
+  "ocaml" {>= "5.0.0"}
+  "dune" {>= "3.17" & >= "3.17"}
+  "core"
+  "ctypes"
+  "ctypes-foreign"
+  "integers"
+  "textutils"
+  "base_quickcheck" {with-test}
+  "expect_test_helpers_core" {with-test}
+  "ppx_jane" {with-test}
```

Comment:
Remember, the `dune-project` file is the source for generating the .opam file. After changing the dune-project file, always remember to run `dune build @default` and make sure there are no updated files. I think you're forgetting to push the changes to this file